source/retryable-writes/retryable-writes.rst

```diff
@@ -31,9 +31,8 @@ write operation(s) and allow for retrying the operation if the driver fails to
 obtain a command result (e.g. network error after a replica set failover). This
 specification will outline how an API for retryable write operations will be
 implemented in drivers. The specification will define an option to enable
-retryable writes when constructing a client session and describe how a
-transaction ID will be provided to write commands executed within that
-client session.
+retryable writes for an application and describe how a transaction ID will be
+provided to write commands executed therein.
```

Comment:
https://github.com/mongodb/specifications/pull/174/commits/ed490e62c9c759f74ae5f30fd3546c9f0bed28d3 renames DriverSession to ClientSession.

https://github.com/mongodb/specifications/pull/174/commits/ea6faf0bdb9d7584bbf68b8a8323b858b0d8aefd removes the "opaque" language.